bootstrap/src/main/config/mediaserver.xml

```diff
@@ -55,7 +55,7 @@
 		<remoteConnection poolSize="50" />
 		<player poolSize="50" />
 		<recorder poolSize="50" />
-		<dtmfDetector poolSize="50" dbi="-35" />
+		<dtmfDetector poolSize="50" dbi="-35" rfc2833Only="false" />
```

Comment:
I'd like to see this configuration moved to a new tag, since we're migrating code to become immutable. The <resources> configuration will likely vanish in the near future.

I'd propose to include a new tag `<dtmf mode="inband|outband|any" dbi="-30">` under the parent tag `<media>`